api/v1alpha1/k8s_types.go

```diff
@@ -103,6 +103,19 @@ type PersistentVolumeClaimSpec struct {
 	VolumeMode *PersistentVolumeMode `json:"volumeMode,omitempty"`
 }
 
+// VolumeSpec contains the Persistent Volume specification for a benchmark run
```

Comment:
The comment `Persistent Volume specification` is not correct, it can be emptyDir which is not persistent.

Additionally, line 82 contains a struct which is a duplication of the Kubernetes API. Moreover, it contains `Size` which is not part of the original specification. We should use `spec.resources.requests.storage` instead.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#persistentvolumeclaimspec-v1-core
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims

Reply:
fixed